src/components/questionary/QuestionaryStepView.tsx

```diff
@@ -99,7 +100,12 @@ export default function QuestionaryStepView(props: {
                             newValue: newValue,
                           },
                         });
-                        handleChange(evt);
+
+                        if (typeof evt === 'string') {
+                          setFieldValue(evt, newValue, true);
```

Comment:
Hi, Peter, I like this part where you using setFieldValue.
I believe we should use setFieldValue always and get rid of handleChange altogether. Just need to change the signature of onComplete. I will do this in a separate pullRequest, so looks good for now.